app/views/repos/_script.html.haml

```diff
@@ -1,9 +1,10 @@
-- if script_url = benchmark_type.script_url
+- if script_url = benchmark.script_url
   %h2.l-align-center
     = t('repos.script_title', graph_name: benchmark_name)
 
-    = link_to benchmark_type.github_url, class: 'btn btn-primary', target: '_link' do
+    = link_to benchmark.github_url, class: 'btn btn-primary', target: '_link' do
       = t('repos.view_github')
 
   - formatter = Rouge::Formatters::HTMLPygments.new(Rouge::Formatters::HTML.new)
+  - require 'net/http' unless defined?(Net::HTTP)
```

Comment:
It would be much better to put it in the controller and/or tests that use this view. You *could* also put it somewhere under config, such as config/initializers, which would guarantee it was *always* loaded. Might be too much, might not. If you only need a thing in the dev environment, you can literally put an "if" around it, such as "if Rails.env.development?"